qa/suites/upgrade/reef-x/stress-split/1-start.yaml

```diff
@@ -21,6 +21,19 @@ overrides:
       - FS_DEGRADED
       - OSDMAP_FLAGS
       - OSD_UPGRADE_FINISHED
+<<<<<<< HEAD
+=======
```

Comment:
It looks like a bogus conflict marker was left behind when cherry picking commit d86a5997798b4436c0f26d6e459a880b88f3841a from `tentacle` to `tentacle-release`.